Routing/RedirectRouteProvider.php

```diff
@@ -49,7 +49,7 @@ public function getRouteCollectionForRequest(Request $request)
         }
 
         $route = new Route(
-            $redirectRoute->getSource(),
+            $pathInfo,
```

Comment:
@nnatter why this change? $redirectRoute->getSource()

Reply:
Think here is missing some of:

```
$redirectRoute = $this->redirectRouteRepository->findEnabledBySource(mb_strtolower($pathInfo), $host)
```

Reply:
Is this the route name? Should we maybe more go with: `'sulu_redirect.redirect_' . $redirectRoute->getId()` or something 🤔

Reply:
no it is the `path` of the route. this is used for generating the regex of the route here:
https://github.com/symfony/symfony/blob/252f85c2c249da7d8f2a490b57cc71199d51fcc9/src/Symfony/Component/Routing/RouteCompiler.php#L75-L92

it looks like the regex is evaluated case sensitive. if we pass `/test123` here but the request contains `TEST123`, symfony will not match the request to the route

Reply:
Okay this make sense not sure if we need to use here `$pathInfo` or `$request->getPathInfo` 🤔